src/bodywork/k8s/auth.py

```diff
@@ -229,7 +229,11 @@ def setup_job_and_deployment_service_accounts(namespace: str) -> None:
                 api_groups=[""],
                 resources=["secrets", "configmaps"],
                 verbs=["get", "list"],
-            )
+            ),
+            k8s.V1PolicyRule(
```

Comment:
I think this needs to go in `setup_workflow_service_account`.

This creates the service accounts that will be used by batch and service stages, so we should not authorise them to be able to interact with anything at the cluster level.

In writing this, I realise how unclear the naming is - sorry.

Reply:
Yeah I got confused here. I am very glad you spotted it. Ive made the change.